app/src/main/java/org/schabi/newpipe/util/StreamItemAdapter.java

```diff
@@ -154,14 +167,15 @@ private View getCustomView(final int position, final View view, final ViewGroup
         if (stream instanceof SubtitlesStream) {
             formatNameView.setText(((SubtitlesStream) stream).getLanguageTag());
         } else {
-            switch (stream.getFormat()) {
-                case WEBMA_OPUS:
+            if (mediaFormat != null) {
+                if (mediaFormat == MediaFormat.WEBMA_OPUS) {
                     // noinspection AndroidLintSetTextI18n
                     formatNameView.setText("opus");
```

Comment:
Well, maybe it would make sense to create a string

Reply:
Probably, don't forget that it's not my code, I just reformatted it.